### PR TITLE
Homework done

### PR DESCRIPTION
**Pull request by Timur85**
# Your Details

Your Name: Timur Abdullaev
- Your City: West Midlands - Class3
- Your Slack Name: Timur Abdullaev

# Homework Details

- Module: HTMl/CSS
- Week: 1

-----
[View rendered .github/pull_request_template.md](https://github.com/Timur85/GitHomeworkFixErrors/blob/main/.github/pull_request_template.md)
[View rendered README.md](https://github.com/Timur85/GitHomeworkFixErrors/blob/main/README.md)
[View rendered homework.md](https://github.com/Timur85/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,39 +4,38 @@
...
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hypertext Markup Language
 
 ## Question 5
 
 What is CSS useful for?
```
Comment (Maria-Chira) [inline homework.md:30]: I think you misunderstood this question. The actual question was what is CSS useful for not what does CSS stand for.

File: homework.md [hunk 2]
```diff
@@ -4,39 +4,38 @@
...
 
-Making a website go faster
+Cascading Style Sheets
```
Comment (Maria-Chira) [inline homework.md:32]: I think you misunderstood this question. The actual question was what is CSS useful for not what does CSS stand for.

File: homework.md [hunk 3]
```diff
@@ -45,36 +44,39 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-When you send a file over the internet
+Pull requests let you tell others about changes you've pushed to a branch in a repository
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It lets you see what you have changed.
```
Comment (Maria-Chira) [inline homework.md:80]: I find this answer a bit confusing. I think you could be a bit more specific.

**Review by Maria-Chira — COMMENTED** [review]
Also on question 15 I believe you will have to double check the answer. By doing a commit we are saving the changes.

File: homework.md (hunk 2 above)
Comment (Timur85) [reply homework.md:32]: Yes, actually, you rightly pointed out, thank you very much.

File: homework.md (hunk 3 above)
Comment (Timur85) [reply homework.md:80]: Many thanks !!!

**Pull request closed without merging**